src/app/layout.tsx

```diff
@@ -22,9 +22,9 @@
  */
 'use client';
 
+import { AuthProvider } from '@/global/utils/auth';
 import { ReactNode } from 'react';
 import { QueryClient, QueryClientProvider } from 'react-query';
-import { AuthProvider } from '@/global/utils/auth';
```

Comment:
nitpick: what's this change's purpose. was it done by https://www.npmjs.com/package/prettier-plugin-organize-imports

Reply:
either that or a merge conflict with the cleanup

Reply:
well, I may suggest adding that package to all projects, for good measure

Reply:
i'm liking the package a lot - very useful

Reply:
works fairly well. wish it also added empty lines between import groups, but this does the job well enough

Reply:
I dislike the empty lines because it's so much a developer preference... loads of places where there's just empty lines for no reason

Reply:
To give a brief example of what I mean to me whitespace emphasises separation of ideas the same way in English we use puntuation in sentences to prevent the "run-on" ones as well as how we use empty lines between separate paragraphs to separate topics etc because if we wrote code the same way I'm writing this it'd still be understandable but god does it take a lot more effort don't you agree?


😂

Reply:
100% agree. unfortunately like most of these topics unless there's a standard way to enforce it , it soon goes to chaos

Reply:
you're right, that's the rub.
I'll keep testing other prettier plugins but the imports order being respected is certainly an improvement. 💯